typing/printtyped.ml

```diff
@@ -449,7 +452,23 @@ and binding_op i ppf x =
     fmt_location x.bop_loc;
   expression i ppf x.bop_exp
 
-and type_parameter i ppf (x, _variance) = core_type i ppf x
+and type_parameter i ppf x =
+  (match x.typa_name.txt with
+   | Some n ->
+     line i ppf "typa_name %a\n" fmt_string_loc { x.typa_name with txt = n }
```

Comment:
You seem fairly careful to show the location here (you introduce `fmt_string_loc` and reconstruct the `string loc` just for this, while it looks like most of the codebase just prints the `.txt` part. Is there a reason for the difference?

Reply:
This comment made me thing more carefully about locations in type parameters, and I think there might be a bug lurking here: by having only a location on typa_name, I think some error messages that complain about a type parameter `'a` might miss the leading `'`.

Reply:
On a closer look, this was fine: it was using the location of the Parsetree's core_type, which includes the `'`. Still, I think this is clearer if the location is separated from the string option. I've pushed a change to this effect.